### ISSUE TITLE
Crash when Broadcast to raid team

### ISSUE THREAD

**Issue by yushicsm** [blueboy/portalclassic#32]
The playerbot was a great project.
but it have many Unstable problem
When i used to be in a raid team
send commands to every bots
sometims there will be crash
Every crashs will Point to “if (!canObeyCommandFrom(fromPlayer))”
line 6113 in PlayerbotAI.cpp
seems the bots cant find where the orders come from

**Comment by cala** [comment]
Hello,

There are a few unstabilities in Playerbot, most due to the fact that there were heavy changes made to the core the last two years and they were only partly ported to Playerbot.

I'm currently trying to prevent this to happen again by officially merging Playerbot into the core: https://github.com/cmangos/mangos-classic/pull/291

To my knowledge, there are three major crashs/unstabilities in Playerbot:
- a crash that happens sometimes right after a NPC is killed with bot engaged in the combat. I highly suspect a loot system issue, but this can also be related to the acquisition of a new target by bots
- a crash that happens when using mage portals to a city when bots are loaded (does not occurs without bots)
- a core stuck that happens when warrior bots get a pull order and at least one more bot is loaded. Again, this is probably related to the acquisition of a new target by bots

These will be investigated and fixed one after another once Playerbot is merged into the core.

However, I did not notice a crash when giving orders to a raid. Can you provide a step-by-step way to reproduce this, please?

**Comment by yushicsm** [comment]
I cant reproduce that because that had a very small chance of happening...

**Comment by yushicsm** [comment]
It’s not the  Playerbot‘s problem
